vignettes/update.Rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: "Using the add/update/remove and adjust functions"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using the update and adjust functions}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  out.width = "100%"
+)
+```
+
+```{r setup, message=FALSE}
+library(epipredict)
+library(recipes)
+```
+
+In this vignette, we will state the main goal of the add/update/remove and adjust functions and describe what part of the processing each function is intended for. We will then demonstrate how to use the sets of add/update/remove functions, followed by the adjust functions, and end with a brief discussion on the tidy methods for recipe and frosting objects. 
```

Comment:
Can you add carriage returns so that this line breaks at 80 characters? Makes reviewing and editing much easier. Same for much of the below.